resources/sql/competitions.sql

```diff
@@ -1,4 +1,4 @@
--- :name competitions-create! :! :n
```

Comment:
Usually when a change is made, you let the original developer make it. So they understand why it changed. If that developer in unavailable, then you can go ahead.

That was a nice catch however.